proto2ros/proto2ros/descriptors/utilities.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+"""
+This module provides utilities to work with Protobuf descriptors.
+
+Many of these utilities trade in terms of paths and locations.
+
+Paths are sequences of numbers that refere to an arbitrarily nested value in a Protobuf message.
```

Comment:
```suggestion
Paths are sequences of numbers that refer to an arbitrarily nested value in a Protobuf message.
```

Reply:
Fixed in https://github.com/bdaiinstitute/ros_utilities/pull/44/commits/ebc60315a6bdfbd9b6ecd0d01f96eeca2b1d8d38.